database/A/Adele/25/Hello

```diff
@@ -84,7 +84,11 @@ It clearly doesn’t tear you apart
 Anymore
 
 
-_____________
-Name    Hello
-Artist  Adele
-Album   Hello
+___________________________________________
+Name              Hello
+Artist            Adele
+Album             25
+Track no          1
+Year              2015
+MusicBrainz ID    0a8e8d55-4b83-4f8a-9732-fbb5ded9f344
+Original text by  Adele Laurie Blue Adkins/Gregory Allen Kurstin
```

Comment:
Not sure if we should make a test to enforce a space between `/`, I usually see it being surrounded by spaces on other sites, but not entirely sure what's correct; I initially intended to use `,  ` (comma + 2 spaces), to make it possible to easily parse list of lyricists, but there's a problem caused by artists and song writers disliking commas, since `/` indicates collaboration, and commas indicate order.

https://www.grammarly.com/blog/slash/
> Many people wonder whether to use a space before and after slashes in a sentence. The answer is typically no on both fronts. Spaces before slashes should be avoided. The only time it’s acceptable to use a space after a slash is when breaking up lines of a poem, song, or play, or when used to separate phrases or multi-word terms for ease of reading.

But not sure if it applies in this case... not a problem to parse it on the website and add extra horizontal margin to slashes though.